challenges/systems/challenge-1.md

```diff
@@ -91,16 +91,14 @@ list of the key observations from the team that affected the result.
 
 ### Testing
 
-Add a couple of high quality tests that cover happy and unhappy scenarios.
-
-Do not try to achieve full test coverage. This will take too long. Take two key
-components, e.g. authentication/authorization layer and networking and implement
-one or two test cases that demonstrate your approach to testing.
+Key components of the challenge (i.e. authorization, output streaming, cgroups) should
+have tests that cover the happy and unhappy scenarious. Do not try to achieve 100% test
```

Comment:
I'm unsure what criteria other folks are reviewing by, but for me, it's less about % coverage and more about seeing multiple testing strategies applied appropriately and effectively. I'd rather see 20% coverage but see examples of table driven testing, unit tests and integration tests, than see 80% coverage using just unit tests.

If other folks are also assessing based on this criteria, then I think it would be helpful to be explicit in stating this within the specification, e.g "Do not try to achieve 100% test coverage - this would take too long - instead try to demonstrate that you can appropriately apply different testing strategies."

Reply:
I'm with Tim here. I don't much care to debate what is a unit test and what is an integration test, I just want to see high quality tests that give me confidence that the code works as intended.

Reply:
Fair enough. I agree it's more about overall efficacy of testing than necessarily "ticking" the box of showing specific testing techniques.